examples/driver.jl

```diff
@@ -0,0 +1,90 @@
+if !haskey(ENV, "BUILDKITE")
```

Comment:
As mentioned in the software meeting, it's better to move (and rename?) this file in the subdirectory(ies) for the hybrid/3D spaces. Please use `git mv` when moving/renaming files so that the git history and contributions are preserved :) 
Thanks!!